src/app/common/components/dataset-list-component/dataset-list-item/dataset-list-item.component.html

```diff
@@ -1,16 +1,26 @@
 <div class="dataset-list-container py-2">
-    <div class="dataset-list-container__title-row">
-        <div class="dataset-list-container__title-row__dataset-name">
-            <i *ngIf="row.kind === 'ROOT'" class="bi bi-journal-album"></i>
-            <mat-icon *ngIf="row.kind === 'DERIVATIVE'">merge</mat-icon>
+    <div class="dataset-list-container__title-row mb-1">
+        <div class="dataset-list-container__title-row__dataset-name d-flex align-items-center">
+            <div class="me-2 mb-1">
+                <a [routerLink]="['/', row.owner.accountName]">
+                    <img
+                        [src]="row.owner.avatarUrl || DEFAULT_AVATAR_URL"
+                        width="20"
+                        height="20"
+                        [alt]="'@' + row.owner.accountName"
+                        [attr.data-test-id]="'user-avatar'"
+                    />
+                </a>
+            </div>
             <a
                 [routerLink]="['/', row.owner.accountName, row.name]"
                 attr.data-test-id="dataset-name-button-{{ rowIndex }}"
                 class="dataset-list-container__title mb-1 pl-0"
             >
                 {{ row.owner.accountName }}/{{ row.name }}
             </a>
-            <app-dataset-visibility class="ms-2" [datasetVisibility]="row.visibility"></app-dataset-visibility>
+            <app-dataset-visibility class="ms-2" [datasetVisibility]="row.visibility" />
+            <app-dataset-kind [kind]="row.kind" />
```

Comment:
Can we make the color of this badge orange for root datasets and blue for derivative, just like in lineage graph:
<img width="1033" height="539" alt="image" src="https://github.com/user-attachments/assets/07a29938-6ebd-49ae-807a-335fd7cb2620" />

I'd like to use the same colors throughout the system to create strong visual associations.